modules/security-context-constraints-about.adoc

```diff
@@ -266,7 +301,8 @@ then this field is considered valid. Note that it is possible that during
 validation, other SCC settings will reject other pod fields and thus cause the
 pod to fail.
 
-*A `SupplementalGroups` SCC strategy of `MustRunAs`*
+[discrete]
+== A `SupplementalGroups` SCC strategy of `MustRunAs`
```

Comment:
I think I'm going to revert these changes, because I don't like how it looks being as a heading now (I meant to put it at level 3, but that still didn't look right). I have another BZ to look at improving the content of these docs (this PR is more for presentation), so I'll have another look at it then.